### PR TITLE
Options Page now has all the state

### PR DESCRIPTION
**Pull request by thebritican**
@zendesk/volunteer 

Second of 3 PRs to finish the Options page 🎉 

All that's left to do is call the filter functions, check their logic, and then hook it up to the URL.

The last PR will also include setting the initial state via the URL.

When navigating to options, we'll just make sure to preserve the query params 😄 

selfie-0

[_GitHub Selfies_](https://github.com/thieman/github-selfies/)

### CODE REVIEW

File: pages/options/index.js [hunk 1]
```diff
@@ -13,6 +13,22 @@ class OptionsPage extends Component {
...
     this.state = {
       taxonomies: [],
+      sortBy: 'name',
+      hours: 'all',
+      services: {
+        food: false,
+        housing: false,
+        hygiene: false,
+        medical: false,
+        technology: false,
```
Comment (craig-day) [inline pages/options/index.js]: I assume these will go away when the URL work is done in the last PR?

File: pages/options/index.js [hunk 2]
```diff
@@ -113,33 +207,36 @@ const ServiceGroup = (props) => (
   </div>
 )
 
+const filterOut = (props, thingsToExclude) => {
+  return Object.keys(props)
+    .filter(prop => !thingsToExclude.includes(prop))
+    .reduce((o, k) => {
+      o[k] = props[k]
+      return o
+    }, {})
```
Comment (craig-day) [inline pages/options/index.js]: 😭

File: pages/options/index.js (hunk 1 above)
Comment (thebritican) [inline pages/options/index.js]: yep

File: pages/options/index.js (hunk 2 above)
Comment (thebritican) [inline pages/options/index.js]: i know. nothin we could do

**Pull request merged**